### ISSUE TITLE
HOW to train

### ISSUE THREAD

**Issue by xiaojidan** [Joker316701882/Salience-Object-Detection#3]
Hi @Joker316701882 
I want to train my own data set, How I can do? Can you share the training codes?

**Comment by Joker316701882** [comment]
@xiaojidan 
Sorry this is confidential so far.  Modified a little according to paper "deeply supervised salient object detection with short connections"

**Comment by Sucran** [comment]
I have another little question in your training if your code is confidential. Did you really set the same learning rate for all layers and use the same data set from the paper  "deeply supervised salient object detection with short connections"？ If so, I'm curious how to fix the overfitting problem in training this model？

**Comment by Joker316701882** [comment]
@Sucran 
Learning rate is set 1e-5 for pixel-level loss all layers all time. And this model combines the encoder-decoder structure mentioned in "Deep Image matting". Train set is MSRA10K. I didn't encounter overfitting problem (random crop is the only augmentation trick applied here). Just left program run one night then got this model. It can be improved if apply more tricks mentioned in these years' segmentation paper.
